python/tvm/relay/op/op_attrs.py

```diff
@@ -16,314 +16,326 @@
 # under the License.
 """The attributes node used for Relay operators"""
 
+import tvm._ffi
 from tvm.ir import Attrs
-from ..base import register_relay_attr_node
 
 
-@register_relay_attr_node
+def _register_relay_attr_node(type_key=None):
+    """Register a Relay attribute node.
+
+    Parameters
+    ----------
+    type_key : str or cls
+        The type key of the node.
+    """
+    return tvm._ffi.register_object(
+        "relay.attrs." + type_key.__name__)(type_key)
+
+
+@_register_relay_attr_node
 class Conv1DAttrs(Attrs):
     """Attributes for nn.conv1d"""
 
 
-@register_relay_attr_node
```

Comment:
consider to use tvm._ffi.register_object directly(and provide the type key), this way FFI navigator can work to help users to navigate across code boundaries

Reply:
ahh, I was lazy to refactor it... Let met do that.